example/src/App.tsx

```diff
@@ -23,18 +23,45 @@ import {
 } from "./Examples";
 import { HomeScreen } from "./Home";
 
+const linking = {
+  config: {
+    screens: {
+      Home: "",
+      Vertices: "vertices",
+      API: "api",
+      Breathe: "breathe",
+      Filters: "filters",
+      Gooey: "gooey",
+      Hue: "hue",
+      Matrix: "matrix",
+      Severance: "severance",
+      Aurora: "aurora",
+      Glassmorphism: "glassmorphism",
+      Neumorphism: "neumorphism",
+      Wallpaper: "wallpaper",
+      Wallet: "wallet",
+      Graphs: "graphs",
+      Animation: "animation",
+      Performance: "performance",
+    },
+  },
+  prefixes: ["rnskia://"],
```

Comment:
This parameter is required by the type definition of the `linking` prop of `NavigationContainer` but it's not used on the web ([doc](https://reactnavigation.org/docs/navigation-container#linkingprefixes))

I've invented the scheme here, but we could also use an empty string. I don't think native deep linking is something relevant for the example project